client/js/components/SignUpForm/SignUp.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const SignUpForm = () => {
+  const [email, setEmail] = React.useState('');
+  const [password, setPassword] = React.useState('');
+
+  //   const {signUp} = React.useContext(AuthContext);
+
+  return (
+    <View>
+      <TextInput
+        placeholder="First Name"
+        value={firstName}
+        onChangeText={setFirstName}
+        secureTextEntry
+      />
+      <TextInput
+        placeholder="Last Name"
+        value={lastName}
+        onChangeText={setLastName}
+        secureTextEntry
+      />
+      <TextInput placeholder="email" value={email} onChangeText={setEmail} />
+      <TextInput
+        placeholder="Password"
+        value={password}
+        onChangeText={setPassword}
+        secureTextEntry
+      />
+      <Button
+        title="Sign Up"
+        onPress={() => signUp({firstName, lastName, email, password})}
```

Comment:
there is no method called signUp in your code. Please define signUp. Make sure when we merge your code it doesn't throw any errors.